pinot-segment-spi/src/main/java/org/apache/pinot/segment/spi/ColumnMetadata.java

```diff
@@ -90,7 +90,23 @@ default boolean isMinMaxValueInvalid() {
   @Nullable
   Set<Integer> getPartitions();
 
-  Map<IndexType<?, ?, ?>, Long> getIndexSizeMap();
+  long getIndexSizeFor(IndexType type);
```

Comment:
Consider adding a default impl for it

Reply:
This seems never used in production code

Reply:
Added default. It's used, for convenience, in a number of old and new tests.